lynx/readme.txt

```diff
@@ -0,0 +1,77 @@
+=== Lynx ===
+Contributors: automattic
+Requires at least: 5.8
+Tested up to: 5.8.3
+Requires PHP: 5.6
+Stable tag: 1.0.1
```

Comment:
I think this can be removed.

Reply:
I'm happy to remove this, but I still see this section in other themes we release. What's the difference? :)

Reply:
The tag isn't required, and I don't think it's meaningful for block themes since they are not so complicated / don't bundle "functionality" like classic themes do, so the latest version should always be considered stable.

Reply:
That makes sense, thanks for taking the time to explain! I updated the readme.txt.